packages/tabs/src/Tabs.ts

```diff
@@ -263,7 +302,7 @@ export class Tabs extends SizedMixin(Focusable, { noDefaultSize: true }) {
         super.updated(changedProperties);
 
         if (changedProperties.has('selected')) {
-            this.scrollToSelection();
+            this.maybeScrollToSelection();
```

Comment:
I like the way you added humour to the names but can we change this to something like checkScrollToSelection() instead?

Reply:
I reverted it to the initial name if that's okay, I think it would be breaking if I change the name of public methods?